src/test/resources/bundles/userCardExamples2/template/en/question.handlebars

```diff
@@ -69,6 +69,14 @@
     }
     loadEntitiesList();
 
+    function loadEntityUsedForResponse() {
+        let responseEntityDiv = document.getElementById("response-entity-div");
+        let responseEntity = templateGateway.getEntityUsedForUserResponse();
+        let userAllowedToRespond = templateGateway.isUserAllowedToRespond();
+        if (userAllowedToRespond && responseEntity) responseEntityDiv.innerHTML = "Entity used by user to respond : " + responseEntity;
+    }
```

Comment:
Show the name of the entity as for entites allowed to respond (use templateGateway.getEntityName)

Reply:
Done